aws/sdk/examples/ec2/src/bin/list-all-instance-events.rs

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+use aws_types::region::ProvideRegion;
+use ec2::{Client, Config, Error, Region, PKG_VERSION};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+struct Opt {
+    /// The AWS Region in which the client is created.
+    #[structopt(short, long)]
+    region: Option<String>,
+
+    /// Whether to display additional runtime information.
+    #[structopt(short, long)]
+    verbose: bool,
+}
+
+/// Shows the scheduled events for the instances in the Region.
+async fn show_events(reg: String) {
+    let region = Region::new(reg.clone());
+    let config = Config::builder().region(region).build();
+    let client = Client::from_conf(config);
+
+    let resp = client.describe_instance_status().send().await;
+
+    println!("Instances in region {}:", reg.clone());
```

Comment:
The clone shouldn't be necessary for `println!` since it will take it by reference.

Reply:
Fixed